AerospikeClient/Cluster/PartitionParser.cs

```diff
@@ -192,8 +194,12 @@ private void DecodeBitmap(Node node, Partitions partitions, int index, int regim
 		{
 			Node[] nodeArray = partitions.replicas[index];
 			int[] regimes = partitions.regimes;
-			char[] chars = Encoding.ASCII.GetChars(buffer, begin, offset - begin);
-			byte[] restoreBuffer = Convert.FromBase64CharArray(chars, 0, chars.Length);
+
+			Span<byte> bufferChars = buffer.AsSpan(start: begin, length: offset - begin);
+			Span<byte> restoreBuffer = stackalloc byte[Base64.GetMaxDecodedFromUtf8Length(bufferChars.Length)];
```

Comment:
This stackalloc is only acceptable if bufferChars is small. Is that the case? Otherwise, an array pool can be introduced.

Reply:
bufferChars is small so stackalloc is fine here.